core/src/main/resources/jenkins/model/Jenkins/404.properties

```diff
@@ -0,0 +1,3 @@
+title = Error: {0}
```

Comment:
😢 the previous message you had in the screenshot looked nicer imo

![image](https://user-images.githubusercontent.com/21194782/134213407-ff24cbd4-27ff-4acb-a008-eac0a26dc9d0.png)